<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gravacao</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-130</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 127);
border-radius: 150px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>490</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 0);
border-radius: 150px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(227, 227, 0);
border-radius: 150px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="bt_toHome">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QAbstractButton{
	
	background-image: url(:/icones/imagens/home.png);
	background-repeat: no-repeat;	
	background-position: center;
	border: 1px solid black;
	border-radius: 20px;
}
QAbstractButton:hover{
background-color: rgb(231, 231, 231);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>-160</x>
       <y>30</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 149, 223);
border-radius: 150px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_gravar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>450</y>
       <width>111</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QAbstractButton{
	background-color: rgb(85, 170, 0);
	border: 2px solid rgb(0, 0, 0);
}
QAbstractButton:hover{
background-color: rgb(98, 197, 0);
}</string>
     </property>
     <property name="text">
      <string>GRAVAR</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>231</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(255, 0, 75);
	border: 0;
	border-radius: 20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="imagem">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>211</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-image: url(:/fotos/imagens/cachorro.png);
	background-repeat: no-repeat;
	border: 0;
	border-radius: 20px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>350</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(255, 0, 75);
	border: 0;
	border-radius: 20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border: 0;
	border-radius: 20px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_frase">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>301</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Frase</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="bt_escutar">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>450</y>
       <width>111</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QAbstractButton{
	background-color:rgb(217, 217, 0);
	border: 2px solid rgb(0, 0, 0);
}
QAbstractButton:hover{
background-color: rgb(240, 240, 0);
}</string>
     </property>
     <property name="text">
      <string>ESCUTAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_enviar">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>450</y>
       <width>111</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QAbstractButton{
	background-color: rgb(231, 0, 115);
	border: 2px solid rgb(0, 0, 0);
}
QAbstractButton:hover{
background-color: rgb(245, 0, 122);
}</string>
     </property>
     <property name="text">
      <string>ENVIAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QAbstractButton{
	background-color: rgb(255, 255, 255);
	border: 1px solid black;
	border-radius: 20px;
}
QAbstractButton:hover{
background-color: rgb(231, 231, 231);
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_direita">
     <property name="geometry">
      <rect>
       <x>657</x>
       <y>370</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	background-image: url(:/icones/imagens/botao-de-pausa.png);
	border: 0;
	border-radius: 12px;
	color: white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_esquerda">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>370</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-image: url(:/icones/botao-de-pausa.png);
	border: 0;
	border-radius: 12px;
	color: white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>530</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 25, 25);
border-radius: 15px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
